Core/clim-core/gadgets.lisp

```diff
@@ -2635,8 +2635,8 @@ if INVOKE-CALLBACK is given."))
           (let* ((topmost-pane (if scroll-p
                                   ;list-pane
                                   (scrolling (:scroll-bar :vertical
-                                              :suggest-height height   ;; Doesn't appear to be working..
-                                              :suggest-width (if scroll-p (+ 30 (bounding-rectangle-width list-pane))))
+                                              :suggested-height height   ;; Doesn't appear to be working..
```

Comment:
Is the comment still true after fixing the typo?

Reply:
I don't know how to verify that (I didn't investigate what the actual code does – I've just picked the change from the original commit). Do you want me to remove the comment without testing?

Reply:
Maybe replace with "TODO investigate whether these arguments actually have an effect"

Reply:
added comment as suggested